chrome/browser/ui/webui/top_chrome/webui_contents_preload_manager.cc

```diff
@@ -83,6 +86,19 @@ class FixedCandidateSelector : public webui::PreloadCandidateSelector {
   GURL webui_url_;
 };
 
+#if BUILDFLAG(ENABLE_SECURE_EMBED)
+content::WebContents* GetSecureEmbedEmbedder(
+    content::BrowserContext* browser_context) {
+  Profile* profile = static_cast<Profile*>(browser_context);
+  if (Browser* browser = chrome::FindLastActiveWithProfile(profile)) {
```

Comment:
This is problematic because the last active browser might not be the browser that opens the side panel.